app/_src/kubernetes-ingress-controller/deployment/k4k8s.md

```diff
@@ -5,76 +5,64 @@ title: Kong for Kubernetes
 Kong for Kubernetes is an Ingress Controller based on the
 Open-Source {{site.base_gateway}}. It consists of two components:
 
-- **Kong**: the Open-Source Gateway
-- **Controller**: a daemon process that integrates with the
-  Kubernetes platform and configures Kong.
+- **Kong** the Open-Source Gateway
+- **Controller** a daemon process that integrates with the Kubernetes platform and configures Kong.
 
 ## Installers
 
 Kong for Kubernetes can be installed using an installer of
-your choice.
+your choice. After the installation is complete, see the [getting started](/kubernetes-ingress-controller/{{page.kong_version}}/guides/getting-started) tutorial to learn more.
 
-Once you've installed Kong for Kubernetes,
-jump to the [next section](#using-kong-for-kubernetes)
-on using it.
+{% navtabs %}
+{% navtab YAML manifests %}
 
-### YAML manifests
-
-Please pick one of the following guides depending on your platform:
+You can install using YAML manifest files in these platforms:
```

Comment:
````suggestion

You can install {{ site.kic_product_name }} with a single `kubectl apply` for testing purposes:

```bash
$ kubectl apply -f https://raw.githubusercontent.com/Kong/kubernetes-ingress-controller/v{{ page.release }}/deploy/single/all-in-one-dbless.yaml
```

There are additional instructions for the following platforms:
````